cli/test/smokehouse/config/config.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/** @type {Record<string, Array<string>>} */
+const exclusions = {
+  // runner: ['excluded1', 'excluded2']
+  // e.g. 'cli': ['a11y']
+  'cli': [],
+  'devtools': [
```

Comment:
```suggestion
  devtools: [
```

Could add to the jsdoc comment L7 the detail that the key is the runner name, but I think it's self evident.

Reply:
Moved the comment to jsDoc.